iib/workers/tasks/build.py

```diff
@@ -514,7 +514,8 @@ def _prepare_request_for_build(
         currently built for; if ``from_index`` is ``None``, then this is used as the list of arches
         to build the index image for
     :param list bundles: the list of bundles to create the bundle mapping on the request
-    :return: a dictionary with the keys: arches, binary_image_resolved, and from_index_resolved.
+    :return: a dictionary with the keys: arches, binary_image_resolved, and from_index_resolved,
+        ocp_version.
```

Comment:
`...binary_image_resolved, and from_index_resolved, ocp_version` => `binary_image_resolved, from_index_resolved, and ocp_version`